syft/messaging/message.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, name, target, args_, kwargs_, return_ids):
         super().__init__()
 
         self.action = ComputationAction(name, target, args_, kwargs_, return_ids)
+        self.return_value = return_value
```

Comment:
What's the type of `return_value` here and what is it for? Should be added to the docstring.